.tekton/manifests/pipeline.yaml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Please rename the file to serviceaccount.yaml

Reply:
Done